README.md

```diff
@@ -69,23 +69,35 @@ Or [download a ZIP of the files](https://github.com/digininja/DVWA/archive/maste
 - [Installing DVWA on Windows using XAMPP](https://youtu.be/Yzksa_WjnY0)
 - [Installing Damn Vulnerable Web Application (DVWA) on Windows 10](https://www.youtube.com/watch?v=cak2lQvBRAo)
 
-### Windows + XAMPP
+### Docker container
```

Comment:
You need to add the following:

- how to access the log files to help with debugging
- tell people how to access the site once it is running
- what happens when they try to run it on port 80 but they already have a web server running on 80 so it won't start up
- does this automatically restart on a reboot or do you need to re-run the docker command?
- if it does automatically restart, how do I stop it from restarting when I want to run Apache on port 80 and it conflicts?